.github/workflows/ci.yml

```diff
@@ -521,8 +521,7 @@ jobs:
   ##############################################################################
   test_shark_model_suite:
```

Comment:
When testing changes to workflows, you could disable all other (non-prerequisite) workflows. CI resources do have costs, so it's nice to limit using them where possible.

Reply:
Yeah I wanted to add general functionality for doing that in a more ergonomic way, but was convinced it wasn't worth it: https://github.com/iree-org/iree/issues/10042. I may revisit that :-P

It looks like ~all you're doing to make this runnable on presubmit here is moving shark to a different workflow file so that when sending a PR you can edit just that file to add the `pull_request` trigger. Is that right, or am I missing something? Separately, this is removing the disabled job to run Shark as part of the postsubmit CI. That was disabled because it was broken in a way that wasn't clearly fixable with the issue to re-enable it https://github.com/iree-org/iree/issues/10653. It looks like it was never re-enabled though. Is running Shark tests on a schedule instead of every commit post-submit the desired behavior? At some point there was a specific request to run it at every commit IIRC. I'm fine making this more of a separate thing. I just want to understand what the goals are here.

Reply:
Maintaining 2 benchmarking infra (perf.iree.dev and SHARK) that run at every commit is a large maintenance burden, especially if we don't own SHARK. The plan is to ensure that perf.iree.dev has enough benchmarks to cover the SHARK set so that there are no surprises in the nightly SHARK run. In cases where developers want to double-check that their changes don't affect SHARK, they have this option of uncommenting `pull_request`.

Reply:
Yup that makes sense to me :-) Just wanted to make sure we're making a conscious decision here